Gordon360/Controllers/CliftonStrengthsController.cs

```diff
@@ -0,0 +1,108 @@
+using Microsoft.AspNetCore.Mvc;
+using Gordon360.Models.CCT.Context;
+using Gordon360.Models.ViewModels;
+using Gordon360.Services;
+using Gordon360.Static.Names;
+using Gordon360.Authorization;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using Gordon360.Models.CCT;
+using Gordon360.Exceptions;
+
+namespace Gordon360.Controllers
+{
+    [Route("api/[controller]")]
+    public class CliftonStrengthsController : GordonControllerBase
+    {
+        private readonly CCTContext _context;
+        private readonly IAccountService _accountsService;
+
+        public CliftonStrengthsController(CCTContext context, IAccountService accountService)
+        {
+            _context = context;
+            _accountsService = accountService;
+        }
+
+        [HttpPost]
+        [Route("")]
+        [StateYourBusiness(operation = Operation.ADD, resource = Resource.CLIFTON_STRENGTHS)]
+        public ActionResult<IEnumerable<CliftonStrengthsUploadResultViewModel>> Post([FromBody] CliftonStrengthsViewModel[] csArr)
+        {
+            IEnumerable<CliftonStrengthsUploadResultViewModel> uploadResults = csArr.Select<CliftonStrengthsViewModel, CliftonStrengthsUploadResultViewModel>(cs =>
```

Comment:
I understand not wanting to use `var`, that's fair enough. The C# Linter literally recommends that we remove the type params for the `Select` though, that's breaking C# conventions. There's no need to explicitly specify the types, and it only hampers readability significantly.

Reply:
Okay, that's fine, thanks for clarifying